apps/app/src/pages/ModernBuild/BuildDetailToolbar.tsx

```diff
@@ -9,65 +10,89 @@ import { memo } from "react";
 import { useBuildDiffFitState } from "./BuildDiffFitState";
 import { useBuildDiffState } from "./BuildDiffState";
 import { useBuildDiffVisibleState } from "./BuildDiffVisibleState";
+import { useBuildHotkey } from "./BuildHotkeys";
 
-const BuildDiffVisibilityToggle = memo(() => {
+const BuildDiffChangesOverlayToggle = memo(() => {
   const { visible, setVisible } = useBuildDiffVisibleState();
+  const toggle = () => setVisible((visible) => !visible);
+  const hotkey = useBuildHotkey("toggleChangesOverlay", toggle, {
+    preventDefault: true,
+  });
   return (
-    <IconButton
-      color="danger"
-      aria-pressed={visible}
-      onClick={() => setVisible((visible) => !visible)}
+    <HotkeyTooltip
+      description={visible ? "Hide changes overlay" : "Show changes overlay"}
+      keys={hotkey.displayKeys}
     >
-      <EyeIcon />
-    </IconButton>
+      <IconButton color="danger" aria-pressed={visible} onClick={toggle}>
+        <EyeIcon />
+      </IconButton>
+    </HotkeyTooltip>
   );
 });
 
 const BuildDiffFitToggle = memo(() => {
   const { contained, setContained } = useBuildDiffFitState();
+  const toggle = () => setContained((contained) => !contained);
+  const hotkey = useBuildHotkey("toggleDiffFit", toggle, {
+    preventDefault: true,
+  });
   return (
-    <IconButton
-      aria-pressed={contained}
-      onClick={() => setContained((contained) => !contained)}
+    <HotkeyTooltip
+      description={contained ? "Expand the screenshot" : "Fit the screenshot"}
```

Comment:
I'm not 100% confident that "Fit the screenshot" is correct sentence. I only know the "fit to something" form for this verb. You maybe should replace "fit" by "contain", "reduce" or "shrink" ?

Reply:
I think it is correct, but if you have a better proposal!

Reply:
If it's ok you, it's ok for me too